cypress/integration/sales/void_a_sales_invoice_then_invoice_the_billing_candidates_again.js

```diff
@@ -0,0 +1,372 @@
+        -1 * totalAmountToPay
+      );
+    });
+  });
+
+  describe('Create second Sales Invoice and check', function() {
+    it('Open the Referenced Billing Candidates', function() {
+      salesOrders.visit(salesOrderRecordId);
+      cy.openReferencedDocuments('C_Order_C_Invoice_Candidate');
+    });
+
+    it('Expect only 1 Billing Candidates row and open it', function() {
+      BillingCandidates.getRows().should('have.length', 1);
+
+      // select the first Table Row and click it (open it)
+      // eslint-disable-next-line prettier/prettier
+      BillingCandidates
+        .getRows()
+        .eq(0)
+        .dblclick();
+    });
+
+    it('Execute action "Generate Invoices" the second time', function() {
+      cy.executeHeaderActionWithDialog('C_Invoice_Candidate_EnqueueSelectionForInvoicing');
+      cy.pressStartButton();
+
+      cy.getNotificationModal(generateInvoicesNotificationModalText);
+      cy.waitUntilProcessIsFinished();
+      cy.getDOMNotificationsNumber().should('equal', 1);
+      // todo check notification inbox text!
+      // expected text:
+      // Rechnung 145808 für Partner G0002 Test Lieferant 1 wurde erstellt.
+      cy.readAllNotifications();
+    });
+
+    it('Expect 3 Sales Invoice and open the third', function() {
+      const whichSI = 2; // 0 indexed
+      const expectedNoRows = 3;
+
+      visitSpecificReferencedSalesInvoice(whichSI, expectedNoRows);
+    });
+
+    it('Check the new Sales Invoice', function() {
+      salesInvoiceChecks(
+        DocumentStatusKey.Completed,
+        salesInvoiceDocumentType,
+        businessPartnerName,
+        false,
+        productName,
+        originalQuantity,
+        totalAmountToPay
+      );
+    });
+  });
+
+  ////////////////////////////////
+  ////////////////////////////////
+  ////////////////////////////////
+  function visitSpecificReferencedSalesInvoice(whichSI, expectedNoRows) {
+    salesOrders.visit(salesOrderRecordId);
+    cy.openReferencedDocuments('AD_RelationType_ID-540160');
+    salesInvoices.getRows().should('have.length', expectedNoRows);
+
+    // eslint-disable-next-line prettier/prettier
+    salesInvoices
+      .getRows()
+      .eq(whichSI)
+      .dblclick();
+  }
+
+  function checkBillingCandidate(qtyDelivered, qtyInvoiced) {
+    cy.expectCheckboxValue('IsToRecompute', false);
+    cy.getStringFieldValue('Bill_BPartner_ID').should('contain', businessPartnerName);
+    cy.getStringFieldValue('C_Order_ID').should('equal', salesOrderDocumentNumber);
+    cy.getStringFieldValue('M_Product_ID').should('contain', productName);
+    cy.getStringFieldValue('QtyEntered').should('equal', originalQuantity.toString(10));
+    cy.getStringFieldValue('QtyOrdered').should('equal', originalQuantity.toString(10));
+    cy.getStringFieldValue('QtyDelivered ').should('equal', qtyDelivered);
+    cy.getStringFieldValue('QtyInvoiced ').should('equal', qtyInvoiced);
+  }
+});
+
+// function clickOnNotificationWithText(text) {
```

Comment:
Can we just remove that ?

Reply:
👍 done in PR https://github.com/metasfresh/metasfresh-e2e/pull/220